src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -539,6 +540,21 @@ def _filter_privacy_request_queryset(
         query = query.filter(PrivacyRequest.id.ilike(f"%{request_id}%"))
     if external_id:
         query = query.filter(PrivacyRequest.external_id.ilike(f"{external_id}%"))
+    if location:
+        # Support filtering by exact location match or country prefix
+        # e.g., "US" matches both "US" and "US-CA", "US-NY", etc.
+        # "US-CA" matches only "US-CA"
+        if "-" in location:
+            # Exact match for subdivision codes
+            query = query.filter(PrivacyRequest.location == location.upper())
+        else:
```

Comment:
We might want to support both `-` and `_` here. `/api/v1/privacy-experience?region=<region>` supports both and I'd prefer to not have one support both and the other only support one.

Reply:
https://github.com/ethyca/fidesplus/blob/main/src/fidesplus/api/service/privacy_experiences/privacy_experience_service.py#L910-L911